themes/app/src/scss/components/_navigation.scss

```diff
@@ -1,16 +1,12 @@
-$menuBorderColor: $grey;
-$menuBorderWidth: 2px;
-$verticalGap: 15px;
-
 .main-nav {
-    border-color: $menuBorderColor;
+    border-color: $sand;
     border-style: solid;
-    border-width: 0 0 $menuBorderWidth;
+    border-width: 0 0 1px;
 }
 
 .main-nav__top-menu {
     align-items: center;
-    padding: $verticalGap 0;
+    padding: 10px 0;
```

Comment:
Can't the constants used previously be changed instead of hard coding them?

Reply:
Nope, there's no point since the padding are different for other elements.